### PR TITLE
samples(generative-ai): Update Grounding Samples and remove Dimensions from MM Embeddings for Video

### PR DESCRIPTION
**Pull request by holtskinner**
- Change Google Search to use GA Endpoint
- Add Variable for VAIS Data Store
- Update models to Gemini 1.5 Flash

### CODE REVIEW

File: generative_ai/gemini_grounding_example.py [hunk 1]
```diff
@@ -59,13 +63,12 @@ def generate_text_with_grounding_vertex_ai_search(
 
     # TODO(developer): Update and un-comment below line
```
Comment (iennae) [inline generative_ai/gemini_grounding_example.py:64]: can you update this comment since it's not just 1 line now but 2?

**Review by iennae — APPROVED** [review]
As long as tests pass, other than the comment update, this LGTM to merge.

**Comment by holtskinner** [comment]
Test failures seem to be unrelated to this sample

**Comment by m-strzelczyk** [comment]
Waaait, that was not what I meant. I didn't want to close this :D

**Review by gericdong — APPROVED** [review]

**Pull request merged**